core/src/main/java/org/infinispan/remoting/transport/jgroups/SingleSiteRequest.java

```diff
@@ -21,6 +22,7 @@
  */
 public class SingleSiteRequest<T> extends AbstractRequest<T> {
    private final String site;
+   private final AtomicBoolean completed = new AtomicBoolean();
```

Comment:
I think it can be removed, too.

Reply:
Same with this one, it is protecting metrics from being out of sync from the actual operation performed.